### PR TITLE
Text to speech

### PR DESCRIPTION
**Pull request by daisymint**

### CODE REVIEW

**Review by aguschin — COMMENTED** [review]
Looks good to me, thanks! I'm not merging because it potentially updates can break the telegram bot. Leaving this up to you @aniervs since you're the only one who can check it works now.

**Comment by aniervs** [comment]
> Looks good to me, thanks! I'm not merging because it potentially updates can break the telegram bot. Leaving this up to you @aniervs since you're the only one who can check it works now.

You see, at this stage, given that we aren't using the saved audios, and we are generating them on the fly, I believe this PR should not be merged.
Right now we're still using Google TTS.
I would recommend keeping this PR open at least until Friday morning, and instead of merging it, create a simpler PR with just the  README of the text2speech_module.

@daisymint @aguschin 

It's just too late in the night for me to try to merge and fix potential issues, although the code seems quite clean.

**Comment by aguschin** [comment]
Totally agree @aniervs. Didn't think you'll work on this right away. It's way too late now.

File: text2speech_module/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### 2. On-the-Fly Audio Generation
+- Generate: Create the audio using the on-the-fly model (pyttsx3) based on the description of the art.
+- Input: Textual description of an art piece
+- Output: Audio file in .wav format sent to the user via the Telegram Bot.
+
+
+## Future Work
+The Tech-to-Speech module is open to contributions, and there are several areas where enhancements could be made:
+1. Audio Retrieval Script: A script to facilitate the retrieval of pre-generated audio from remote storage could streamline the process and enhance the user experience.
+2. Model Fine-Tuning: Fine-tuning the underlying TTS models could ensure better and more consistent audio quality.
```
Comment (aguschin) [inline text2speech_module/README.md:39]:
```suggestion
2. Model Fine-Tuning: Fine-tuning the underlying TTS models could ensure better and more consistent audio quality!
```

**Review by aguschin — APPROVED** [review]

**Pull request merged**